libs/ballot-interpreter/src/hmpb-ts/cli.ts

```diff
@@ -516,14 +552,41 @@ export async function main(args: string[]): Promise<number> {
       json,
       scoreWriteIns,
       debug,
+      useDefaultMarkThresholds,
     });
   }
 
-  if (electionDefinitionPath && ballotPathSideA && ballotPathSideB) {
+  if (
+    electionDefinitionPath &&
+    systemSettingsPath &&
+    ballotPathSideA &&
+    ballotPathSideB
+  ) {
+    const parseElectionDefinitionResult = safeParseElectionDefinition(
+      await fs.readFile(electionDefinitionPath, 'utf8')
+    );
+
+    if (parseElectionDefinitionResult.isErr()) {
+      stderr.write(
+        `Error parsing election definition: ${
+          parseElectionDefinitionResult.err().message
+        }\n`
+      );
+      usage(stderr);
+      return 1;
+    }
+
+    const electionDefinition = parseElectionDefinitionResult.ok();
+
+    const systemSettings = safeParseSystemSettings(
+      await fs.readFile(systemSettingsPath, 'utf8')
+    ).ok();
```

Comment:
Should we apply the same error handling that we do for election definition parsing if we fail to parse the system settings file? Looks like we're currently silently ignoring failures to parse the system settings file

Reply:
Nice catch, thanks. Added a warning and explanatory comment: 1619cfbe8